### ISSUE TITLE
REAL minimum specs to evaluate?

### ISSUE THREAD

**Issue by syadnom** [irontec/ivozprovider#268]
The docs says 4 cores, 4GB, and 30GB storage, what about actual minimums to test with just a trunk and a few devices?  1 core and 1GB ram do it?  I know freeswitch and kamailio can run on something pretty light, but mysql and php not so much.  Don't want to spend a bunch of cash on an expensive VM to test with.

**Comment by syadnom** [comment]
oh, for a stand alone system...

**Comment by cruzccl** [comment]
Hi @syadnom 

Sincerely we have not tried running IvozProvider in VMs with such minimal hardware resources, even our development machines have 4vCPUs and 2GB RAM, but we would highly appreciate if you tried and tell us .

Kamailio, rtpproxy and rtpengine can run on something light but Asterisk makes quite use of MySQL and PHP in AGI execution.

Nevertheless, bear in mind that there are 3 parameters to take into account:

- **Number of registered UACs**: they generate periodical REGISTER transactions that are handled by Kamailio and stored in MySQL from time to time. Registry expiration should be kept high to avoid unnecessary traffic.

- **Number of concurrent calls**: once a call is established between endpoints, the media-relay (rtpproxy/rtpengine) gets loaded handling its media.

- **Number of calls per seconds**: call establishment is heavy for Asterisk, that needs to instantiate some php AGIs (that make some SQL queries) to know how to handle it.

To sum up, those resource should be enough to make some testing but production environments should be distributed in several much more powerful machines to squeeze IvozProvider strenghts.

Regards,
